ansible_collections/arista/avd/roles/eos_designs/doc/node-types-v4.0.md

```diff
@@ -59,6 +62,9 @@ node_type_keys:
       vtep_ip: <path to J2 template - default inherited from templates.ip_addressing.vtep_ip >
 
     # Optional | Override interface_descriptions templates
+    # Path is relative to playbook
+    # See: https://avd.sh/en/stable/roles/eos_cli_config_gen/index.html#extensibility-with-custom-templates
```

Comment:
@pvinci-arista, thanks for taking this on and helping improve the documentation.
IMO we should explain here, and not just have a link to eos_cli_config_gen documentation!
It is ok in the documentation if we repeat ourselves a little!